rules/community/microsoft/windows/hacktool_purpleknight_execution.yaral

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+rule hacktool_purpleknight_execution {
+
+    meta:
+      author = "Georg Lauenstein - suresecure GmbH"
+      description = "This detection rule identifies the execution of the Purple Knight tool, a free Active Directory security assessment utility developed by Semperis. Purple Knight is designed to scan for AD vulnerabilities, misconfigurations, and common attack paths. While it is a legitimate tool used by defenders, its execution in production environments may also indicate red team activity or unauthorized reconnaissance by adversaries attempting to map domain weaknesses."
+      rule_id = "mr_69e428bb-c946-4e3f-abb8-0e28316f2389"
+      rule_name = "Purple Knight Tool Execution Detected"
+      tactic = "TA0007"
+      technique = "T1087, T1069, T1018"
```

Comment:
Can you please choose a _single_ MITRE ATT&CK technique that best aligns with these rules?